docs/server/settings.rst

```diff
@@ -281,9 +281,13 @@ Translation environment configuration settings.
 
 .. setting:: POOTLE_TM_SERVER
 
-.. versionadded:: 2.7
```

Comment:
I'm not sure you want to lose that.  Without it yu don't know when this feature actually arrived.